qt/python/mantidqt/widgets/instrumentview/test/test_instrumentview_io.py

```diff
@@ -0,0 +1,124 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2017 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantidqt package
+#
+from __future__ import (absolute_import, division, print_function, unicode_literals)
+
+from mantidqt.project.encoderfactory import EncoderFactory
+from mantidqt.project.decoderfactory import DecoderFactory
+from mantidqt.widgets.instrumentview.io import InstrumentViewEncoder, InstrumentViewDecoder
+from mantid.simpleapi import CreateSampleWorkspace
+from mantidqt.utils.qt.test import GuiTest
+
+import os
+
+if os.name == 'nt':
+    INSTRUMENT_VIEW_DICT = {u'workspaceName': u'ws',
```

Comment:
The is a different output for windows expected, because that's what the output seems to be on that OS. The value from the dict is set as expected however at the time of decoding it has changed already. It seems to be a Windows exclusive issue.

Reply:
@Pasarus I can't reproduce the problem on Windows. I removed the `if os.name == 'nt'` block on my laptop and the test passes. Can you remove it and resubmit to see what the builders say (I have made a small change to the buildscript on master to remove old MantidPlot settings)

Reply:
Ok will do